requirements-build.txt

```diff
@@ -0,0 +1,40 @@
+# These are top-level requirements that are needed to build the local project,
+# but which we do not wish to export when we copy over the minimal manifest.
+
+
+
+pytest==6.2.4
+attrs==21.2.0
+iniconfig==1.1.1
+packaging==20.9
+pluggy==0.13.1
+py==1.10.0
+pyparsing==2.4.7
+toml==0.10.2
+
+pylint==2.8.3
+astroid==2.5.6
+isort==5.9.1
+lazy-object-proxy==1.6.0
+mccabe==0.6.1
+toml==0.10.2
+wrapt==1.12.1
+
+pylint-protobuf==0.13
+astroid==2.5.6
+isort==5.9.1
+lazy-object-proxy==1.6.0
+mccabe==0.6.1
+pylint==2.8.3
+toml==0.10.2
+wrapt==1.12.1
```

Comment:
You probably don't need pylint-protobuf and its dependencies
Also you probably don't need pytesseracy or timeout-decorator either.